packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -732,6 +732,21 @@ export type CallAdapterLocator =
   | /* @conditional-compile-remove(rooms) */ RoomCallLocator
   | /* @conditional-compile-remove(teams-adhoc-call) */ /* @conditional-compile-remove(PSTN-calls) */ CallParticipantsLocator;
 
+/* @conditional-compile-remove(rooms) */
```

Comment:
nit: Define the type without any conditional compilation so you can use it internally.

It is already conditionally exported from `./index.tsx`, as it should be.